experimental/cloudstrapper/playbooks/roles/docker-platform/tasks/magma-repo.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: create magma directory
+  file:
+    path: "/opt/magma"
+    state: directory
+    mode: u+rwx
+    owner: "{{ usrName }}"
+  become: true
+
+- name: download github repo
```

Comment:
The agw_install_docker script appears to remove anything in the /opt/magma directory and clone it. I agree that doing it here makes sense too, but we might have to refactor it.

Reply:
Yes best would be to just checkout that script! I will add the install of docker in this script in this PR too.

Reply:
might be best to transform that docker intstall script in an ansible role later to avoid checking out the repo here